client/flutter/lib/l10n/intl_pt.arb

```diff
@@ -0,0 +1,26 @@
+{
+  "worldHealthOrganization": "Organização Mundial da Saúde",
+  "learnMore": "Saiba mais",
+  "protectYourself": "Proteja-se",
+  "checkYourHealth": "Cheque sua saúde",
```

Comment:
If this sentence will be used for a title or menu link, maybe it is more correct to use the word "Verifique" instead of "Cheque".
- Verifique (to Verify)

The word "Cheque" is mostly used for bank checks.

Other words that may be useful:
- Examine (to scan)
- Veja (to look)
- Observe (to observe)
```suggestion
  "checkYourHealth": "Verifique sua saúde",
```

I prefer "Verifique (to verify)" or "Examine (to scan)". But it depends of context.